docs/attributes-registry/client.md

```diff
@@ -15,9 +15,35 @@ This also covers UDP network interactions where one side initiates the interacti
 | Attribute  | Type | Description  | Examples  |
 |---|---|---|---|
 | `client.address` | string | ![Stable](https://img.shields.io/badge/-stable-lightgreen)<br>Client address - domain name if available without reverse DNS lookup; otherwise, IP address or Unix domain socket name. [1] | `client.example.com`; `10.1.2.80`; `/tmp/my.sock` |
-| `client.port` | int | ![Stable](https://img.shields.io/badge/-stable-lightgreen)<br>Client port number. [2] | `65123` |
+| `client.geo.city_name` | string | City name. | `Montreal`; `Berlin` |
```

Comment:
Do we expect any client.geo and server.geo attributes to be present simultaneously?  I thought the purpose of a global attribute registry is to represent attributes that mean the same wherever they are used, and that suits well for the geo.* attributes. That is, geo.city_name means the same whether it's used in the context of client or server or any other that's described at the end at https://www.elastic.co/guide/en/ecs/current/ecs-geo.html

Even if the client.geo.x and server.geo.x attributes are required in the same object, I think this problem is unsolved currently - for eg., one cannot have two `url.full` attributes in the same object if I have to represent the referring url or the url being redirected to in the case of 302.

I think we should omit the `client` and `server` prefix.

Reply:
> Even if the client.geo.x and server.geo.x attributes are required in the same object, I think this problem is unsolved currently - for eg., one cannot have two url.full attributes in the same object if I have to represent the referring url or the url being redirected to in the case of 302.

@scheler  This is exactly what we are trying to solve with https://github.com/open-telemetry/semantic-conventions/issues/339. For the geo example that means:
- we will define `geo.*` fields only once in the registry
- and then, `geo.*` fields will be (explicitly) reusable under different namespaces (while keeping their original definition just applied into the context of the target namespace). This will result in `client.geo.*` and `server.geo.*` anyways. 
- this will also solve the problem of having the same fields on a single signal with different context (as they would have different namespaces)

> I think we should omit the client and server prefix.

- We are just defining it here explicitly (for now) because we are still working on the [reuse concept](https://github.com/open-telemetry/semantic-conventions/issues/339) for semantic conventions and don't want to block Geo fields because of that. 
- If you mean that we should not have a prefix for geo at all (even with the reuse concept), then I don't fully get the reasoning, yet. As described above it solves a concrete problem we have today, that you also mentioned yourself with being able to have the same group of attributes twice on a signal with different context. 
- If we would omit the `client`, `server` prefixes for geo, we would break ECS with no reason (no actual conflict with semconv or attribute naming guidelines), which has been explicitly stated as something to avoid whenever possible [in the original OTEP](https://github.com/open-telemetry/oteps/pull/222).

Reply:
@AlexanderWert thanks for the clarification. One issue though is that one cannot use `geo.city_name` itself. Is that intended here?

Reply:
@scheler Yes it's intended. In [Elastic common schema](https://www.elastic.co/guide/en/ecs/current/ecs-geo.html), `geo` fields are not expected to be used directly at the root of the events.